Develop mobile branch

ghermaico135 opened the pull request:
mobile portfolio both work section and  about us

ghermaico135 committed:
adding of html and styling of main mobile responsive

ghermaico135 committed:
adding of index file and styling of about mylife  section

ghermaico135 committed:
Resolved merge conflict by incorporating both suggestions.

ghermaico135 committed:
adjusting white styling space

ghermaico135 committed:
removing css selectors

ghermaico135 committed:
removing css selectors

ghermaico135 commented:
Responsive mobile portfolio consisting of work section and about by implementing flex and grids

mukhammadrizooff commented on index.html:
```diff
@@ -9,8 +9,13 @@
+=======
   <div class="container">
     <!-- Header  -->
+>>>>>>> main
```
- [ ] [REQUIRED] Please remove `Github Conflict` symbols and give the correct style to your `Headed` page like [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=VtDtYBmX3HDlKmDq-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/228895402-7de331e4-1f06-481f-af9e-f5ab9dfff224.png)

mukhammadrizooff commented on index.html:
```diff
@@ -72,9 +129,258 @@ <h2 class="greeting">
+</header>
+      <!----------------------------------------End of Header section--------------------------- -->
+      <!-- --------------------------------------Start of Main and Work section 1--------------------------------->
+      <main>
+      <div class="work-section">
```
- [ ] [REQUIRED] Please implement the correct font size to your `headings` and give the correct style to your `button`🙏🏼

![image](https://user-images.githubusercontent.com/63915024/228899105-81e11ad0-7d1a-40ae-9378-9ab9e0f25008.png)

mukhammadrizooff commented on index.html:
```diff
@@ -72,9 +129,258 @@ <h2 class="greeting">
+
+  <!---------------------------------------------footer ------------------------------------------------------- -->
+  <footer>
+    <section>
+        <div class="about-section">
```
- [ ] [REQUIRED] Good job so far! Please give this section a proper design because some `buttons` are missing and the design of the `elements` is not like [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=VtDtYBmX3HDlKmDq-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/228899961-d66ec665-5bd9-475e-9410-977c1f41d6de.png)

mukhammadrizooff reviewed (changes requested):
Hi @ghermaico135, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media0.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ghermaico135 committed:
adding modified index.html and styling for the review

ghermaico135 committed:
clearing white space from style.css

ghermaico135 commented:
I made change in the header section happened due to git conflict
I changed the font size of the h1 and align the items
I included the icons lost due to git conflict and changed the button size according figma
I remove the background of the grid items.
I hopefully i made all the changed required by the reviewer

iambenkis commented on index.html:
```diff
@@ -36,45 +38,308 @@ <h2 class="greeting">
+                            <h5>CSS</h5>
+                          </div>
+                        </div>
+
+                        <div class="grid separate">
+                              <div class="soft-skill">
+                                <h5 class="language">FrameWorks</h5>
+                                <img src="./images/Icon/CodingLine.png" alt="code cheveron">
+                              </div>
+
+                              <div class="soft-skill">
+                                  <h5 class="language">Skills</h5>
+                                  <img src="./images/Icon/CodingLine.png" alt="code cheveron">
```
- [ ] As requested by the previous code review , it could be better to make sure that the style of these Items stick to the Figma design 👍. I really appreciate the effort you put in it but make sure you improve the border styling 🤙 

![Capture](https://user-images.githubusercontent.com/80458199/229095570-64237624-7b1f-4a10-ba7c-3859c3ddf9b4.PNG)

iambenkis commented on CSS/style.css:
```diff
@@ -18,23 +20,27 @@ body {
   background-repeat: no-repeat;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-bottom: 1.2rem;
```
- [ ] [**OPTIONAL**] Kindly make sure this header section fill all the width of the screen, as it is represented in the Figma design 

![Capture](https://user-images.githubusercontent.com/80458199/229096434-72519fab-5b75-4f75-b509-0fbf1ade7517.PNG)

iambenkis reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ghermaico135  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ghermaico135 committed:
added the changes on the width and removing border box on the stylin

ghermaico135 commented:
I made the changed required by the reviwer both width of the screen and bottom border

AbelG101 commented on index.html:
```diff
@@ -9,25 +9,27 @@
+                <h1 class="greeting">
+                  I'm Michael.
+                  Glad to see you!
+                </h1>
```
- [ ] [OPTIONAL] As suggested by the previous reviewer kindly make sure you've fixed the negative space around the headline of your page as it's making your portfolio look unprofessional 👶:
    ![image](https://user-images.githubusercontent.com/104968236/229109899-2e020044-7255-4726-a370-fc3e23706f76.png)
- As I've highlighted in yellow the negative spacing isn't shown in the Figma design so I kindly suggest you fix it.
> * [ ]  [**OPTIONAL**] Kindly make sure this header section fill all the width of the screen, as it is represented in the Figma design
> 
> 
> ![Capture](https://user-images.githubusercontent.com/80458199/229096434-72519fab-5b75-4f75-b509-0fbf1ade7517.PNG)

AbelG101 commented on CSS/style.css:
```diff
@@ -152,8 +174,279 @@ ul {
+  color: #396df2;
+}
+
+.btn {
+  width: 200px;
+  height: 55px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 0.5rem;
+  padding: 1rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  text-align: center;
+  text-decoration: none;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+}
```
- [ ] [OPTIONAL] Please make sure the "See project" button looks similar to the given [FIgma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=t45uZqimwMP5BSMR-0) as I've noticed the width & padding are not In sync with the design.
    - Your design:
       ![firefox_pGgtdol9H1](https://user-images.githubusercontent.com/104968236/229111042-48276b0f-f0dd-434b-b436-e2464283eba9.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/229111196-b8797edd-1d95-4e8f-88c0-9d24d86d2d42.png)
- [ ] [OPTIONAL] Also please make sure you've added the `:hover` and `:active` state to the buttons as shown in the Figma design:
    - Here's the design as a reminder:
       ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/229111513-7f3f7f23-ec7d-46ce-a3cc-29c897da2a40.png)

AbelG101 reviewed (approved):
![](https://media.giphy.com/media/eKfovWZOlOpbTDEX9F/giphy.gif)

# Hi @ghermaico135 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ghermaico135 committed:
final changes after approving on the button background color during hover and removing negative space
